geopaparazzispatialitelibrary/src/eu/geopaparazzi/spatialite/database/spatial/core/daos/DaoSpatialite.java

```diff
@@ -436,9 +439,51 @@ public static void updateFeatureGeometry(String id, Geometry geometry, String ge
         sbIn.append(SpatialiteUtilities.SPATIALTABLE_ID_FIELD).append("=");
         sbIn.append(id);
         String insertQuery = sbIn.toString();
-        database.exec(insertQuery, null);
+        dbSpatialite.exec(insertQuery, null);
+    }
+    public static double[] longLat2Srid(Database dbSpatialite, double lon, double lat, String s_srid)  throws Exception {
```

Comment:
Ah, ok, now I get it. Lat long can be transformed to that layer's srid. Ok, nice.